my-app/src/pages/Discover.js

```diff
@@ -1,12 +1,240 @@
+                <div className="discover-button-container">
+                    <div>
+                        <h1>
+                            Discover
+                        </h1>
+                    </div>
+
+                    {/* New Post Button */}
+                    <div>
+                        <button className="new-post-button">+</button>
+                    </div>
+                </div>   
+
+                {/* import 2 users' names from API to display */}
+                {displayedUsers.map((user, index) => (
+                    <SocialMediaPost 
+                        key={user.id} 
+                        user={user} 
+                        index={index} 
+                    />
+                ))}
+            </div>
+
+            {/* right side with browsing new people or places to visit */}
+            <div className = "social-media-col">
+                <h1>
+                    Places For You
+                </h1>
+
+                {/* contains the boba sgop recommendations */}
+                <div className = 'boba-shop-container'>
+                    {/* boba shop 1 */}
+                    <div className = 'boba-shop-image-container'>
+                        {/* shop image */}
+                        <img src="https://i.imgur.com/iEvMlCc.png" alt="Smoked Oreo Smoothie Promo from Happy Lemon Promotion"/>
+
+                        {/* shop name */}
+                        <div className = 'boba-shop-name-text'>
+                            <p2>
+                                Happy Lemon
+                            </p2>
+                        </div>
+
+                    </div>
+
+                    {/* boba shop 2 */}
+                    <div className = 'boba-shop-image-container'>
+                        {/* shop image */}
+                        <img src="https://i.imgur.com/0ig5N81.png" alt="Rose Drinks from Ume Tea Promotion"/>
+
+                        {/* shop name */}
+                         <div className = 'boba-shop-name-text'>
+                            <p2>
+                                Ume Tea
+                            </p2>
+                        </div>
+                        
+                    </div>
+
+                    {/* boba shop 3*/}
+                    <div className = 'boba-shop-image-container'>
+                        {/* shop image */}
+                        <img src="https://i.imgur.com/tstytZI.png" alt="Two Drinks From Gathers"/>
+
+                         {/* shop name */}
+                         <div className = 'boba-shop-name-text'>
+                            <p2>
+                                Gathers
+                            </p2>
+                        </div>
+                        
+                    </div>
+                </div>
+                
+            </div>
+        </div>
     </div>
     )
+}
+
+//
+// external functions
+//
+
+
```

Comment:
Highly recommend putting the functions below here in separate files so this one file doesn't get too big